tests/test_validation.py

```diff
@@ -62,3 +65,27 @@ def test_datetime_validation_failure():
     payload = {"upload_date": "test o'clock"}
     with pytest.raises(EventSchemaValidationError):
         _ = get_validated_payload(payload=payload, schema=UploadDateModel)
+
+
+@pytest.mark.parametrize(
+    "payload,assertions",
+    [
+        ({}, ["upload_date", "type=missing"]),
+        ({"upload_date": True}, ["type=string", "should be a valid string"]),
+    ],
+)
+def test_error_messages(payload: dict[str, Any], assertions: list[str]):
+    """Verify that the pydantic error includes information as expected."""
+    version_text = f"'UploadDateModel' from ghga-event-schemas v{__version__}"
+    payload_text = f"The complete payload was: {json.dumps(payload)}"
+
+    with pytest.raises(EventSchemaValidationError) as error:
+        get_validated_payload(payload, UploadDateModel)
+
+    error_text = str(error.value.args[0])
+
+    for string in assertions:
+        assert string in error_text
+    assert "upload_date" in error_text
```

Comment:
this can be removed, maybe added in the second test params

Reply:
wait, sorry, the schema is fixed in the test anyway, so this can be also fixed

alternatively, make the schema part of the params